source/extensions/tracers/opentelemetry/span_context_extractor.h

```diff
@@ -15,8 +15,8 @@ namespace OpenTelemetry {
 
 class OpenTelemetryConstantValues {
 public:
-  const Tracing::TraceContextHandler TRACE_PARENT{"traceparent"};
```

Comment:
It would be better to extract this into configuration, we can upstream it if we do

Reply:
Ok let me create a ticker for that.